README.md

```diff
@@ -2,7 +2,7 @@ Table of Contents
 =================
```

Comment:
Should we also add some README lines for the embedded dashboard? At least to inform consumers that we provide an embedded dashboard as well, I believe I did not see it in the OOTB dashboard alternatives